modules/server/src/gqlServer.ts

```diff
@@ -0,0 +1,24 @@
+import { Client } from '@elastic/elasticsearch';
+import { GraphQLResolveInfo } from 'graphql';
+
+export type Context = {
+	esClient: Client;
```

Comment:
TODO: we will need to abstract this into  an ESClient provider, so users can switch seamlessly between ES v7 and v8